x-pack/plugins/viz_editor/public/common/components/operation_editor/operation_definitions.tsx

```diff
@@ -0,0 +1,431 @@
+  const operation = column as WindowOperation;
+  const { argument } = operation;
+  const opDefinition = getOperationDefinition(column.operation);
+  const options = opDefinition
+    .applicableFields(visModel.datasource.fields, props)
+    .map((f: DatasourceField) => ({
+      value: f.name,
+      text: f.name,
+    }));
+  const calculations = [
+    { value: 'max', text: 'Max' },
+    { value: 'min', text: 'Min' },
+    { value: 'sum', text: 'Sum' },
+    { value: 'stdDev', text: 'Standard Deviation' },
+    { value: 'unweightedAvg', text: 'Unweighted Average' },
+    { value: 'linearWeightedAvg', text: 'Linear Weighted Average' },
+    { value: 'ewma', text: 'Ewma' },
+    { value: 'holt', text: 'Holt' },
+    { value: 'holtWinters', text: 'Holt Winters' },
+  ];
+
+  return (
+    <EuiForm>
+      <EuiFormRow label="Field">
+        <EuiSelect
+          options={options}
+          value={argument && argument.field}
+          onChange={e =>
+            onColumnChange({
+              ...operation,
+              argument: {
+                ...argument,
+                field: e.target.value,
+              },
+            })
+          }
+          aria-label="Field"
+        />
+      </EuiFormRow>
+      <EuiFormRow label="Calculation">
+        <EuiSelect
+          options={calculations}
+          value={argument && argument.windowFunction}
+          onChange={e =>
+            onColumnChange({
+              ...operation,
+              argument: {
+                ...argument,
+                windowFunction: e.target.value as WindowFunction,
+              },
+            })
+          }
+          aria-label="Window Function"
+        />
+      </EuiFormRow>
+      <EuiFormRow label="Window size">
+        <EuiFieldNumber
+          value={argument && argument.windowSize}
+          onChange={e =>
+            onColumnChange({
+              ...operation,
+              argument: {
+                ...argument,
+                windowSize: parseInt(e.target.value, 10),
+              },
+            })
+          }
+          aria-label="Window size"
+        />
+      </EuiFormRow>
+    </EuiForm>
+  );
+}
+
+export const operations: OperationDefinition[] = [
+  {
+    name: 'Values',
+    type: 'column',
+    applicableFields: (fields, { allowedColumnTypes }) =>
+      fields.filter(({ type }) => (allowedColumnTypes ? allowedColumnTypes.includes(type) : true)),
+    editor: fieldOperationEditor,
+    toSelectClause(
+      currentOperation: SelectOperation | undefined,
+      fields: DatasourceField[]
+    ): ColumnOperation {
+      return {
+        operation: 'column',
+        alias: getFieldName(currentOperation, fields),
+        argument: { field: getFieldName(currentOperation, fields) },
+      };
+    },
+    summarize(op: ColumnOperation) {
+      return (
+        <span>
+          <EuiIcon type="string" className="configPanel-summary-icon" />
+          {`Values of ${op.argument.field}`}
```

Comment:
There isn't yet a helper function that generates labels for operations on fields, but what I've been doing is `${operationToName('column')} of ${op.argument.field}` so that it's at least easier to replace with an i18n version later

Reply:
Makes sense, thanks.